js/common.js

```diff
@@ -1,24 +1,50 @@
 'use strict';
 
+const head = document['head'] || document.getElementsByTagName("head")[0] || root;
 const header = $('#header');
+const $body = $('body');
 const initialHeadHeight = header.innerHeight();
 const tocNav = $('#toc');
 const headerFixPosition = $('.nav-hero-container').innerHeight();
+const topDocNavHeight = $('.top-doc-nav-container').innerHeight();
 const stickyElement = $('.sticky-element');
 const stickyElementPosition = headerFixPosition; // Sticky element top-offset (154px)
 const goTopBtn = $('#go-top-btn');
 const copyrightEl = $('.copyright');
-const isFaqPage = $('body').is('.faq');
+const $pre = $('pre');
+const isFaqPage = $body.is('.faq');
+const isDocsPage = $body.is('.docs');
+const isPromoPage = $body.is('.promo-page');
 const topOffset = 12; // Offset from the `header` navigation
 const scrollToOffset = initialHeadHeight + topOffset;
 
+/** Code color variables */
+const $colorSelector = $('.color-selector');
+const $colorSelectorLinks = $('.color-selector .color-link');
+const $selectorDark = $('.color-link.dark');
+const $selectorLight = $('.color-link.light');
+const colorDark = 'dark';
+const colorLight = 'light';
+const cookieColorName = 'themeColor';
+const baseStylesUrl = '/libs/prettify/skins/';
```

Comment:
I would say it is not  `baseStylesUrl`.  It is code wrapper skins url or something similar.